src/test/java/duke/StorageTest.java

```diff
@@ -0,0 +1,16 @@
+package duke;
+
+import org.junit.jupiter.api.Test;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.fail;
+
+
+public class StorageTest {
+
+    @Test
```

Comment:
maybe you could consider testing some other cases such as checking whether todo, deadline and event are correctly stored in the storage?